### DISCUSSION TITLE
z80pack web site

### DISCUSSION THREAD

**Discussion by udo-munk** [udo-munk/z80pack#440] [Q&A]
The z80pack web site is not available at Autometer anymore, private stuff needed to be removed.
For now it is available here: https://www.icl1900.co.uk/unix4fun/

**Comment by ravn** [comment]
I would suggest writing a small README.md file in the root of the project, containing a link with a small explanatory note to the web site, and directions about how to get up and running on the supported platforms.   

This might be as simple as just pointing to the installation guide.  I can do a small initial version if interested.

It might also be helpful to create github actions to let Github do continuous integration using these.

**Comment by udo-munk** [comment]
Sure, any contributions are welcome, be it documentation, bug fixes, improvement... Just create a PR with your stuff.

**Comment by ravn** [comment]
I have created https://github.com/udo-munk/z80pack/pull/441 with an initial version plus enabling Github Actions build.
